cmd/subnetcmd/deploy.go

```diff
@@ -298,13 +310,40 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// get keys for blockchain tx signing
+	if subnetAuthKeys != nil {
+		if err := checkSubnetAuthKeys(subnetAuthKeys, controlKeys, threshold); err != nil {
+			return err
+		}
+	}
+	if subnetAuthKeys == nil {
+		subnetAuthKeys, err = getSubnetAuthKeys(controlKeys, threshold)
+		if err != nil {
+			return err
+		}
+	}
+	ux.Logger.PrintToUser("Your subnet auth keys for chain creation: %s", subnetAuthKeys)
+
 	// deploy to public network
 	deployer := subnet.NewPublicDeployer(app, useLedger, kc, network)
-	subnetID, blockchainID, err := deployer.Deploy(controlKeys, threshold, chain, chainGenesis)
+	subnetID, blockchainDeployed, blockchainID, tx, err := deployer.Deploy(controlKeys, subnetAuthKeys, threshold, chain, chainGenesis)
```

Comment:
Is the old path preserved if threshold == 1?

Reply:
yes

Reply:
i really coded taking that into account and testing both paths.
besides that, we have the simulated fuji e2e